app/src/main/java/de/danoeh/antennapod/fragment/CoverFragment.java

```diff
@@ -33,22 +33,15 @@
 
     public static CoverFragment newInstance(Playable item) {
         CoverFragment f = new CoverFragment();
-        if (item != null) {
-            Bundle args = new Bundle();
-            args.putParcelable(ARG_PLAYABLE, item);
-            f.setArguments(args);
-        }
+        f.media = item;
```

Comment:
My main reason to do it was because it wasn't the most convenient way of handling media changes.
So what do you suggest? Change the arguments every time we change the media, just so android can properly recover the fragment? But then the fragment does not have an id, so I think Android does not save its instance.